std_execution.bs

```diff
@@ -2641,13 +2641,33 @@ namespace std::execution {
   template&lt;class T>
     concept <i>class-type</i> = <i>decays-to</i>&lt;T, T> && is_class_v&lt;T>;  // exposition only
 
+  // [execution.queries], general queries
+  inline namespace <i>unspecified</i> {
+    struct get_scheduler_t;
+    inline constexpr get_scheduler_t get_scheduler{};
+    struct get_allocator_t;
+    inline constexpr get_allocator_t get_allocator{};
+    struct get_stop_token_t;
+    inline constexpr get_stop_token_t get_stop_token{};
+  }
+  template <class T>
+    using stop_token_of_t =
+      remove_cvref_t<decltype(get_stop_token(declval<T>()))>;
+
   // [execution.schedulers], schedulers
   template&lt;class S>
     concept scheduler = <i>see-below</i>;
 
   // [execution.schedulers.queries], scheduler queries
   enum class forward_progress_guarantee;
   inline namespace <i>unspecified</i> {
+    struct forwarding_scheduler_query_t;
+    inline constexpr forwarding_scheduler_query_t forwarding_scheduler_query{};
+
+    template &lt;class T>
+      concept <i>forwarding-scheduler-query</i> = // exposition only
+        requires(T t) { requires forwarding_scheduler_query(t); };
+
```

Comment:
I do not see an implementation of forwarding scheduler query. Did I miss it?

Reply:
I haven't implemented any scheduler queries yet.

Reply:
I would expect the forwarding cpo to be implemented, and used by scheduler implementations to forward cpos, even if there are no cpos to forward in the proposal.